apache/spark
# [SPARK-43769][CONNECT] Implement 'levenshtein(str1, str2[, threshold])' functions

### What changes were proposed in this pull request?
The pr aims to implement 'levenshtein(str1, str2[, threshold])' functions for `connect` module.

### Why are the changes needed?
After [Add a max distance argument to the levenshtein() function](https://issues.apache.org/jira/browse/SPARK-43493) We have already implemented it on the scala side, so we need to align it.


### Does this PR introduce _any_ user-facing change?
Yes, new API for Connect.

### How was this patch tested?
- Pass GA.
- Manual testing
1../build/sbt "connect-client-jvm/testOnly *ClientE2ETestSuite*"
2.sh dev/connect-jvm-client-mima-check
